pom.xml

```diff
@@ -80,6 +80,9 @@
         <version.server.bom>24.0.0.Beta1-SNAPSHOT</version.server.bom>
         <version.microprofile.bom>24.0.0.Beta1-SNAPSHOT</version.microprofile.bom>
 
+        <version.server.bootable-jar>23.0.0.Final</version.server.bootable-jar>
```

Comment:
can you please move this to the quickstart pom.xml, as microprofile Quickstarts have

Reply:
Why must each quickstarts that uses bootable jar redefines this version?
How is it different from quickstarts sharing the same `version.microprofile.bom`?

Reply:
I'll revert this change but I don't understand the guiding principles here. When do we share things between quickstarts?

Reply:
At some point BOMs will move to each quickstart too, the future direction is that the parent is just an aggregator, and each QS is as much independent as possible. There are always issues due to the parent dependency, for instance wrt s2i testing and codeready workspaces.

Reply:
ok, thanks for the explanation